build-system/dep-check-config.js

```diff
@@ -72,10 +72,16 @@ exports.rules = [
     filesMatching: 'ads/**/*.js',
     mustNotDependOn: 'src/**/*.js',
     whitelist: [
+      'ads/**->src/document-state.js',
+      'ads/**->src/experiments.js',
       'ads/**->src/log.js',
+      'ads/**->src/mode.js',
+      'ads/**->src/timer.js',
```

Comment:
Should we be whitelisting these for everything under ads, or only for specific modules?  (More specifically, the mode.js whitelist here subsumes the one for traffic-experiments.js only, below.)